scheduler/src/cook/compute_cluster.clj

```diff
@@ -530,11 +530,13 @@
         dissoc-ca-cert-vals
         (fn [configs]
           (map-vals dissoc-ca-cert configs))]
-    (log/info "Updating dynamic clusters."
-              {:current-configs (dissoc-ca-cert-vals current-configs)
-               :force? force?
-               :new-configs (dissoc-ca-cert-vals new-configs)
-               :updates (map #(update % :goal-config dissoc-ca-cert) updates)})
+    (log/info "Updating dynamic clusters.")
+    (doseq [current-config (dissoc-ca-cert-vals current-configs)]
+      (log/info "Updating dynamic cluster" (key current-config) "current config:" current-config))
```

Comment:
current config and new config look to be logging the map entry, so adding the specific key is redundant, but harmless.

Reply:
I like the format of updating cluster [cluster], shame we can't match it on the updated configs section.